ts/Series/Treegraph/TreegraphSeries.ts

```diff
@@ -532,40 +532,49 @@ class TreegraphSeries extends TreemapSeries {
         state?: StatesOptionsKey
     ): SVGAttributes {
         const series = this,
-            levelOptions =
+            levelOptions = point &&
```

Comment:
Now that we have a situation where `point` may be undefined, the typing must reflect it:

https://github.com/highcharts/highcharts/blob/a05042ac5298b093986527f166a9b92e548e472d/ts/Series/Treegraph/TreegraphSeries.ts#L530-L533

Like in the base series, where the `point` argument is optional:

https://github.com/highcharts/highcharts/blob/a05042ac5298b093986527f166a9b92e548e472d/ts/Core/Series/Series.ts#L2811-L2814